src/main/java/uk/gov/pay/directdebit/payments/services/SandboxService.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.pay.directdebit.payments.services;
+
+import uk.gov.pay.directdebit.payers.model.Payer;
+import uk.gov.pay.directdebit.payments.model.DirectDebitPaymentProvider;
+
+public class SandboxService implements DirectDebitPaymentProvider {
+
+
+    public SandboxService() {
+    }
+
+
+    @Override
+    public String createCustomer(String paymentRequestExternalId, Payer payer, String sortCode, String accountNumber) {
+        return null;
```

Comment:
This is an interface segregation. Is fine returning a service but this must be a top level one. From our point of view we are interested on payers and customers are a Gocardless thing so I for me I would see:
A real DirectDebitService has a method to createPayer then two impls Gocardless that actually creates a payer and a customer and Sandbox that only creates a payer. So the interface is not segregated.This means to discover the provider at top level (as soon the resource is hit)